test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -133,7 +134,8 @@ describe('CaseTypeTab', () => {
       // 'SolicitorCoRespondent', - TODO - uncomment this when we go live with AOS pack 2
       'serviceApplication',
       'LinkedCase',
-      'Language'
+      'Language',
+      'ConfidentialDocuments'
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.